tms/clickup/views.py

```diff
@@ -19,3 +21,44 @@ def post(self, request):
         return JsonResponse({'token': token.key,
                              "user": user.username,
                              "message": "Successfully logged in."}, status=status.HTTP_200_OK)
+
+
+class ProjectCreateView(ListCreateAPIView):
+    queryset = Project.objects.all()
+    serializer_class = ProjectSerializer
+
+    def post(self, request, *args, **kwargs):
+        serializer = self.get_serializer(data=request.data)
+        serializer.is_valid(raise_exception=True)
+        self.perform_create(serializer)
+        return JsonResponse({"message": "Project created successfully",
+                             "data": serializer.validated_data}, status=status.HTTP_201_CREATED)
+
+
+class ProjectListView(ListAPIView):
+    queryset = Project.objects.all()
+    serializer_class = ProjectSerializer
+
```

Comment:
why did you write this class in labove list create api view you can pass queryset and listing view also work